### DISCUSSION TITLE
Is there a way to fetch Azure resource health details using promitor agent scraper?

### DISCUSSION THREAD

**Discussion by adhiraj-g** [tomkerkhove/promitor#1632] [Q&A] [answered]
Can we use promitor agent scraper to fetch out resource health details?

From what i read, the agent scraper checks Azure Monitor API. As for resource health, there is a separate API.
https://docs.microsoft.com/en-us/rest/api/resourcehealth/

Just wanted to confirm whether promitor agent scraper can fetch out resource health API as well? Or is there any seperate scraper which needs to be added?

**Comment by tomkerkhove** [comment, marked as answer]
We do not support Azure Resource Health yet, but curious what your expected behavior would be?

There are no metrics available, but eager to learn what metrics you would be expecting.

https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported

**Reply by adhiraj-g** [reply]
Hi Tom,

For Azure Resource Health, it would be to fetch out availability status of azure resources. Support to fetch and monitor across multiple subscriptions.

https://docs.microsoft.com/en-us/rest/api/resourcehealth/availabilitystatuses

While searching for alternatives, there is an exporter available for this purpose https://github.com/FXinnovation/azure-health-exporter.

**Reply by tomkerkhove** [reply]
We could consider adding support for it, I'm just not sure what the output would look like? 🤔

**Reply by tomkerkhove** [reply]
Thanks for the tip on the exporter!
